Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "REopt"
 uuid = "d36ad4e8-d74a-4f7a-ace1-eaea049febf6"
-authors = ["Nick Laws <nick.laws@nrel.gov>"]
-version = "0.16.2"
+authors = ["Nick Laws <nick.laws@nrel.gov>", "Hallie Dunham <hallie.dunham@nrel.gov>", "Bill Becker <william.becker@nrel.gov>", "Bhavesh Rathod <william.becker@nrel.gov>", "Alex Zolan <alexander.aolan@nrel.gov>", "Amanda Farthing <amanda.farthing@nrel.gov>"]
```

Comment:
Seems like my email is incorrect. For emails, should we move Nick's email out or towards the end of this list now?

Reply:
updated your email in https://github.com/NREL/REopt.jl/pull/100/commits/12498dd5c145826d281f4f659365cfda48c5484a. In my opinion it's fine (and most accurate currently) to leave Nick's name at the front, but we should probably remove his email because it's no longer active.